Duplicati/Library/Backend/AzureBlob/AzureBlobBackend.cs

```diff
@@ -21,19 +21,25 @@
 using Duplicati.Library.Interface;
 using System.Collections.Generic;
 using System.IO;
-using System.Threading;
-using System.Threading.Tasks;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace Duplicati.Library.Backend.AzureBlob
 {
+    // ReSharper disable once UnusedMember.Global
+    // This class is instantiated dynamically in the BackendLoader.
     public class AzureBlobBackend : IStreamingBackend
     {
         private readonly AzureBlobWrapper _azureBlob;
 
+        // ReSharper disable once UnusedMember.Global
+        // This constructor is needed by the BackendLoader.
         public AzureBlobBackend()
         {
         }
 
+        // ReSharper disable once UnusedMember.Global
+        // This constructor is needed by the BackendLoader.
```

Comment:
I know many people don't use ReSharper or the Rider IDE.  Let me know if you prefer not to see these types of comments that disable warnings.